packages/phoenix-event-display/src/managers/three-manager/import-manager.ts

```diff
@@ -245,7 +245,10 @@ export class ImportManager {
                 menuName + geoName.substring(0, pos).replace('_>_', ' > ');
               geoName = geoName.substring(pos + 3);
             }
-            this.processGeometry(geometry, geoName, scale);
+            var opacity = undefined;
+            if ('opacity' in geometry.userData)
+              opacity = geometry.userData['opacity'];
+            this.processGeometry(geometry, geoName, scale, undefined, opacity);
```

Comment:
Since the `geometry` is already passed to the function, I don't think we need to do the calculation here. Plus there's a simpler way.